### DISCUSSION TITLE
Proposal: list dashboards that using this

### DISCUSSION THREAD

**Discussion by nikolaydubina** [cncf/landscape2#511] [Ideas]
Hi, come here from HackerNews. Found this by ML infra landscape. https://ai-infra.fun/?group=default&view-mode=grid

Now I wonder, what other landscapes people built? I my self want to built and announce one this in couple days myself. Should we list somewhere usecases for people to discover? (this is akin "meta-awesomelist of awesome lists", or can just add in bottom of README of this project.)

Nice work! Very impressive!

**Comment by nikolaydubina** [comment]
haha found it https://github.com/cncf/landscape2-sites#landscape2-sites

**Comment by tegioz** [comment]
Thanks @nikolaydubina!

You can find some more we maintain [here](https://github.com/cncf/landscape2-sites?tab=readme-ov-file#landscape2-sites), but there are some more in the wild 🙂

We were planning to add an ADOPTERS file to list as many as we could find, maybe you'd like to list yours on it!

**Reply by nikolaydubina** [reply]
definitely interested listing community drivel landscapes! once again, nice work! 👍🏻

**Reply by tegioz** [reply]
https://github.com/cncf/landscape2/blob/main/ADOPTERS.md 🙂
